packages/containers/src/DeleteResource/sagas.js

```diff
@@ -75,17 +74,26 @@ export function* deleteResourceValidate(
 			yield put({
 				type: deleteResourceConst.DIALOG_BOX_DELETE_RESOURCE_SUCCESS,
 				model: {
+					id: safeId,
 					labelResource: resource.get('label') || resource.get('name', ''),
 				},
 			});
+			yield put(
+				cmf.actions.collections.mutate(resourceLocator, {
+					delete: [safeId],
+				}),
+			);
```

Comment:
That's a good idea but it won't word on CQRS based applications.
Under that architecture we have to wait for an acknowledgment from the back-end in order to actually remove the element. That's why we've kept the DELETE request and the collection removal steps separated.

Reply:
ok, thank you for this info. then i will remove this part.